### PR TITLE
release notes for 3.7.0

### PR DESCRIPTION
**Pull request by vasanthsubram**

### CODE REVIEW

File: docs/source/releasenotes/add-ons/3.0.rst [hunk 1]
```diff
@@ -34,7 +34,9 @@ Bug Fixes
 
 - [KC-6380] - CSV Parser Incorrectly Parsing Pipe-delimited file
 
-
+3.0.6
+^^^^^
+- [KC-5588] - Update S3 Source To Work Without Files In A Directory
```
Comment (bmcrea) [inline docs/source/releasenotes/add-ons/3.0.rst:39]: This isn't a bug fix it's an improvement/new feature

Comment (etmarx) [reply docs/source/releasenotes/add-ons/3.0.rst:39]: Brenna, I thought we decided to just listing the stories in the addon release underneath the addon release number. What are you expecting to see here?

Comment (bmcrea) [reply docs/source/releasenotes/add-ons/3.0.rst:39]: I thought we did too but up above it says "Bug Fixes". I would remove the words "Bug fix" and move the Azure store down to the 3.0.6 list.

File: docs/source/releasenotes/3.7.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+3.7 Release Notes
+==================
+
+New Features
+------------
+- [KC-6398] - Azure Blob Storage Source/Sink
+- [KC-6405] - Job input output record count
+- [KC-6428] - Response codes for queries with multiple data sets
+
+Bug Fixes
```
Comment (etmarx) [inline docs/source/releasenotes/3.7.rst:12]: Please move Bug Fixes underneath - Add-ons version: 3.0.6

File: docs/source/releasenotes/add-ons/3.0.rst (hunk 1 above)
Comment (vasanthsubram) [reply docs/source/releasenotes/add-ons/3.0.rst:39]: moved the KC-5588 to the new feature under the 3.7 release notes - should it be removed from the addons page?

Comment (bmcrea) [reply docs/source/releasenotes/add-ons/3.0.rst:39]: No they should both be on the addons page as new features under 3.0.6.

Also please update the title to: Update S3 Source to no longer require folder and add region as a field.

The old title wasn't very accurate. I should have updated it before.

**Review by etmarx — APPROVED** [review]

File: docs/source/releasenotes/3.7.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+3.7 Release Notes
+==================
+
+New Features
+------------
+- [KC-6398] - Azure Blob Storage Source/Sink
+- [KC-6405] - Job input output record count
+- [KC-6428] - Response codes for queries with multiple data sets
+- [KC-5588] - Update S3 Source To Work Without Files In A Directory
```
Comment (bmcrea) [inline docs/source/releasenotes/3.7.rst]: Please put these in numerical order by ticket number.

**Review by bmcrea — APPROVED** [review]

**Pull request merged**